doc/changelog.rst

```diff
@@ -2,6 +2,11 @@
 Changelog
 =========
 
+- :release:`0.5.0 <2020.05.27>`
```

Comment:
Are we releasing tonight? Or should we make it 05.31 and have some time to wrap it up?

Reply:
I don't really mind. We can keep updating this date until we actually release.